ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.spec.ts

```diff
@@ -76,14 +78,22 @@ describe('AddressEmbeddedComponent', () => {
         return of(null);
       }
     });
+    const translateServiceSpy: jasmine.SpyObj<NGXTranslateService> = jasmine.createSpyObj('NGXTranslateService', ['getTranslation']);
+    // @ts-ignore
+    translateServiceSpy.getTranslation.and.callFake((word: string | Array<string>, keyToValue?: object) => {
+      return of(Array.isArray(word) ?
+          word.map((each, i) => ({ each: 'label' + i })).reduce((prev, current) => ({ ...prev, ...current }), {}) as object :
+          'label1');
+    });
 
     TestBed.configureTestingModule({
       declarations: [ AddressEmbeddedComponent, FakeAddressInputComponent, ValidationMessage ],
       providers: [
         {provide: AddressService, useValue: addressServiceSpy},
-        {provide: SubmitAnnouncementService, useValue: submitAnnounceService}
+        {provide: SubmitAnnouncementService, useValue: submitAnnounceService},
+        {provide: NGXTranslateService, useValue: translateServiceSpy}
       ],
-      imports: [MatCardModule, MatRadioModule, ReactiveFormsModule]
+      imports: [MatCardModule, MatRadioModule, ReactiveFormsModule, TranslateTestingModule]
     })
```

Comment:
To keep tests running needed to mock translate directive, pipe, and service. Did not write tests for their use though.